src/themes/creator.js

```diff
@@ -157,6 +157,7 @@ export default function createTheme(
       linkText: primitives.primary400,
       linkVisited: primitives.primary500,
       linkHover: primitives.primary600,
+      linkActive: primitives.primary400,
```

Comment:
Can you change this to `primary600`?

Reply:
Should I also update this in dark theme?